{{cookiecutter.project_slug}}/Pipfile

```
@@ -36,7 +36,7 @@ test = "py.test --ff --cov {{cookiecutter.project_slug}}"
 check-fixtures = "py.test {{cookiecutter.project_slug}} --dead-fixtures --dup-fixtures"
 check-yapf = "yapf -d -r -p {{cookiecutter.project_slug}}"
 fix-yapf = "yapf -i -r -p {{cookiecutter.project_slug}}"
-check-isort = "isort -rc -c {{cookiecutter.project_slug}}"
+check-isort = "isort -c {{cookiecutter.project_slug}}"
 fix-isort = "isort -rc {{cookiecutter.project_slug}}"
```

```suggestion
fix-isort = "isort {{cookiecutter.project_slug}}"
```

Maybe we should update the fix step too, don't ya think?